README.md

```diff
@@ -260,6 +260,8 @@ that caused Neoformat to be invoked.
   - [`mix format`](https://hexdocs.pm/mix/master/Mix.Tasks.Format.html)
 - Elm
   - [`elm-format`](https://github.com/avh4/elm-format)
+- Erlang
+  - [`erlfmt`](https://github.com/WhatsApp/erlfmt)
```

Comment:
could you also update the `docs/neoformat.txt`?

Otherwise looks good!

Reply:
Certainly, added 8060c27